addons/stock/stock.py

```diff
@@ -4512,6 +4517,7 @@ def split_quantities(self, cr, uid, ids, context=None):
             if pack.product_qty - pack.qty_done > 0.0 and pack.qty_done < pack.product_qty:
                 pack2 = self.copy(cr, uid, pack.id, default={'qty_done': 0.0, 'product_qty': pack.product_qty - pack.qty_done}, context=context)
                 self.write(cr, uid, [pack.id], {'product_qty': pack.qty_done}, context=context)
+                self._copy_remaining_pack_lot_ids(cr, uid, pack.id, pack2, context=context)
```

Comment:
I mean the signature of the _copy_remaining_pack_lot_ids function

On Fri, Sep 9, 2016 at 8:19 AM, Nicolas Martinelli <notifications@github.com
> wrote:

> In addons/stock/stock.py
> <https://github.com/odoo/odoo/pull/13278#discussion_r78133468>:
>
> > @@ -4512,6 +4517,7 @@ def split_quantities(self, cr, uid, ids, context=None):
> >              if pack.product_qty - pack.qty_done > 0.0 and pack.qty_done < pack.product_qty:
> >                  pack2 = self.copy(cr, uid, pack.id, default={'qty_done': 0.0, 'product_qty': pack.product_qty - pack.qty_done}, context=context)
> >                  self.write(cr, uid, [pack.id], {'product_qty': pack.qty_done}, context=context)
> > +                self._copy_remaining_pack_lot_ids(cr, uid, pack.id, pack2, context=context)
>
> @jco-odoo <https://github.com/jco-odoo> : yes, only one element since we
> are in a loop.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/odoo/odoo/pull/13278/files/994f71fbc96d7e8119207264e7b7e447aba62aab#r78133468>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADp7e3lfoR_KSTICyc5sayO-O9-6gTOvks5qoPpigaJpZM4Jr3ui>
> .
>